src/js/ui/footer/index.js

```diff
@@ -12,9 +12,9 @@ export const footer = () => {
       <div class="col mb-5 mb-md-0" data-footer-col>
         <h4 class="text-white mb-3 fw-bold">For Companies</h4>
         <ul class="p-0 d-flex flex-column gap-1">
-        <li class="list-group-item"><a href="#" class="nav-link text-white" target="_blanc" id="navlink-footer-id">About Noroff Jobs</a></li>
-        <li class="list-group-item"><a href="#" class="nav-link text-white" target="_blanc" id="navlink-footer-id">Company User Guide</a></li>
-        <li class="list-group-item"><a href="#" class="nav-link text-white" target="_blanc" id="navlink-footer-id">FAQ</a></li>
+        <li class="list-group-item"><a href="#" class="nav-link text-white navlink-footer-id" target="_blanc" >About Noroff Jobs</a></li>
```

Comment:
Please correct the typo in target attributes

Reply:
ok will do

Reply:
Why is this marked as resolved when there still is a typo in target attributes ?
Current is "target="_blanc"
should be; "target="_blank"

This goes for all places where this might occure

Reply:
![Skjermbilde 2024-01-17 143404](https://github.com/NoroffFEU/agency.noroff.dev/assets/114483893/4b943186-6c1e-4c70-8755-7713d6c614d2)


done